services/opencollective/opencollective-base.js

```diff
@@ -1,26 +1,30 @@
+import gql from 'graphql-tag'
 import Joi from 'joi'
+import { BaseGraphqlService } from '../index.js'
 import { nonNegativeInteger } from '../validators.js'
-import { BaseJsonService } from '../index.js'
 import { metric } from '../text-formatters.js'
 
-// https://developer.opencollective.com/#/api/collectives?id=get-info
-const collectiveDetailsSchema = Joi.object().keys({
-  slug: Joi.string().required(),
-  backersCount: nonNegativeInteger,
-})
+const schema = Joi.object({
+  data: Joi.object({
+    account: Joi.object({
+      name: Joi.string().required(),
+      slug: Joi.string().required(),
```

Comment:
Looks like there is some stuff defined in the schema that we are not actually using. For example, we've made `name` and `slug` required fields in the response, but I don't think we're actually using them anywhere? Can we make sure we only fail if fields we rely on are missing. There's some docs on writing schemas at https://github.com/badges/shields/blob/master/doc/input-validation.md